src/main/java/com/google/api/codegen/config/ResourceNameMessageConfig.java

```diff
@@ -41,6 +42,17 @@ public static ResourceNameMessageConfig createResourceNameMessageConfig(
     return new AutoValue_ResourceNameMessageConfig(fullyQualifiedMessageName, fieldEntityMap);
   }
 
+  @Nullable
+  public static ResourceNameMessageConfig createResourceNameMessageConfig(Field field) {
```

Comment:
This never returns null right?

I'm not very familiar with resource names. Why do we make the assumption on L49?

Reply:
Removed the `@Nullable`. That was some copypasta.

Hmm thinking about it now, we don't have to make that assumption. Removed the comment.